client/login.py

```diff
@@ -18,7 +18,7 @@ def loginPage(screen):
     while not done:
         for event in pg.event.get():
             if event.type == pg.QUIT:
-                sys.exit(0)
+                return SHOULD_STOP_GAME
```

Comment:
this is incorrect because once an exit command is called (simply press x on the client once you are in the login page), see what happens with the logic in main. the loginPage function also returns an address. If you naively return a boolean, the rest of the logic is set to fail. Please consider how to account for both of these possibilities (loginPage could return an address string or a boolean flag to stop the game) in main and proceed with implementing it